### DISCUSSION TITLE
Can I join #Team

### DISCUSSION THREAD

**Discussion by DESTROYER3264** [ROM-builders/temporary#17779] [Q&A]
Greetings sir. I have been already using ROM-builders and wants to join team.

Device tree- https://github.com/DESTROYER3264/device_motorola_liber
Vendor tree- https://github.com/DESTROYER3264/vendor_motorola_liber
kernel tree- https://github.com/DESTROYER3264/kernel_motorola_liber

Successful builds
https://cirrus-ci.com/build/5789454653194240
https://cirrus-ci.com/build/6613568347439104

**Comment by Apon77** [comment]
please check #team instruction in telegram

**Reply by DESTROYER3264** [reply]
Yeah I have checked instructions and applied only after that
